app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -137,4 +136,16 @@ class PrescriptionRepositoryAndroidTest {
     assertThat(softDeletedPrescription.createdAt).isEqualTo(prescription.createdAt)
     assertThat(softDeletedPrescription.syncStatus).isEqualTo(SyncStatus.PENDING)
   }
+
+  @Test
+  fun facilities_should_be_overridable() {
+    val prescription = testData.prescription(name = "Churro")
```

Comment:
I want a doctor who prescribes churros for high BP too.

Reply:
😄